config/environments/development.rb

```diff
@@ -106,8 +106,8 @@ def yarn_integrity_enabled?
   config.logger = ActiveSupport::TaggedLogging.new(logger)
 
   config.after_initialize do
-    Bullet.enable = true
-    Bullet.console = true
+    Bullet.enable = false
```

Comment:
Did you mean to turn these off?

Reply:
Yeah. I can probably come up with a different solution, but this thing changed where the script gets automatically inserted into the page, which affected CSS. I'm sure we could figure out something else though. Happy to get this worked out before shipping anything.